### PR TITLE
Add minipad

### PR DESCRIPTION
**Pull request by minisbett**

### CODE REVIEW

File: org/minipad/index.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: org
+title: minipad
```
Comment (tannewt) [inline org/minipad/index.md:3]: This should be `minisbett`

**Review by tannewt — CHANGES_REQUESTED** [review]
Firmware looks ok. Please link to the hardware source files from the software repo's readme.

File: org/minipad/index.md (hunk 1 above)
Comment (minisbett) [reply org/minipad/index.md:3]: but isn't it supposed to be the organization name?

**Comment by minisbett** [comment]
> Firmware looks ok. Please link to the hardware source files from the software repo's readme.

Im afraid I cannot do that yet as the hardware files are not ready to be published to my community yet. It will be ready once the configuration software is done, which requires us to use this PID for the prototype testers.

Another question: Is a CC license being recognized as an OSS/OSHW license?

**Comment by tannewt** [comment]
> > Firmware looks ok. Please link to the hardware source files from the software repo's readme.
> 
> Im afraid I cannot do that yet as the hardware files are not ready to be published to my community yet. It will be ready once the configuration software is done, which requires us to use this PID for the prototype testers.

Please use a pid.codes, testing PID until you are ready.

> 
> Another question: Is a CC license being recognized as an OSS/OSHW license?

Some are. I follow OSHWA's guidance: https://www.oshwa.org/definition/

File: org/minipad/index.md (hunk 1 above)
Comment (minisbett) [reply org/minipad/index.md:3]: You mean the github organization name? Then it's "minipadkb".

Comment (tannewt) [reply org/minipad/index.md:3]: It doesn't need to match your github organization. The pid's `owner` needs to match the org folder name.

**Comment by minisbett** [comment]
I'd like to re-open this. Both the firmware and hardware files are now public.
https://github.com/minipadkb/minipad-firmware
https://github.com/minipadkb/minipad

**Pull request closed without merging**